### PR TITLE
Added support for wrapped message serialization and deserialization.

### PR DESCRIPTION
**Pull request by rvdovychenko-sift**
Added separate config option for the payload field name.
Added source support.
Added related tests.

### CODE REVIEW

**Review by hxu-sift — COMMENTED** [review]
Several potential action items for the next step:
- For further testing, if you have access to Kafka instances, we can test the package with Kafka connect clusters.
- If we're not contributing back to the original repo now, then we need to ask CI to have a CI/CD process for uploading the jar file to GCS.
- If we want to contribute back, then we should start with [opening an issue](https://github.com/blueapron/kafka-connect-protobuf-converter#contributing) in their repo and describe our added functionality and use cases.

File: src/main/java/com/blueapron/connect/protobuf/ProtobufData.java [hunk 1]
```diff
@@ -50,6 +50,10 @@
   private final Schema schema;
   private final String legacyName;
   private final boolean useConnectSchemaMap;
+  // For this PoC, when this is set to true, we assume the byte string field is the actual payload
```
[the remark this answers is not in the document]
Comment (rvdovychenko-sift) [reply src/main/java/com/blueapron/connect/protobuf/ProtobufData.java]: Dropped extra comments.

File: src/main/java/com/blueapron/connect/protobuf/ProtobufData.java [hunk 2]
```diff
@@ -106,11 +136,31 @@ private String getProtoFieldName(String descriptorForTypeName, String connectFie
     this.schema = toConnectSchema(getBuilder().getDefaultInstanceForType());
   }
 
+  private SchemaAndValue wrappedToConnectData(Message message) {
+    // Build schema for payload message
+    final Schema payloadSchema = toConnectSchema(getPayloadBuilder().getDefaultInstanceForType());
+
+    // Find the BYTES field from the wrapper message
```
[the remark this answers is not in the document]
Comment (rvdovychenko-sift) [reply src/main/java/com/blueapron/connect/protobuf/ProtobufData.java]: Dropped extra comments.

**Review by hxu-sift — APPROVED** [review]
LGTM

**Pull request merged**